lib/car.rb

```diff
@@ -0,0 +1,31 @@
+class Car
+  attr_reader :name, :monthly_payment, :loan_length
+
+  def initialize(name, monthly_payment, loan_length)
+    @name = name
+    @monthly_payment = monthly_payment
+    @loan_length = loan_length
+  end
+
+  def make
+    @name = @name.split(" ")
+    return @name[0]
+  end
+
+  def model
+    @name = @name.split(" ")
+    return @name[1]
+  end
+
+  def total_cost
+    return loan_length * monthly_payment
+  end
+
+  # def color
+  #   @color = nil
+  # end
+  #
+  # def paint!
+  # end
```

Comment:
appreciate that you started these. What I would do for color is to initialize it to nil within the initializer so when you call `.color` before painting it returns `nil`

```
def initialize(name, monthly_payment, loan_length)
    @name = name
    ....
    @color = nil
  end
```

then you can declare the `paint!` function to take in 1 parameter and update your instance variable:

```
def paint!(color)
  @color = color
end
```